src/approval/approval.service.ts

```diff
@@ -1,101 +1,111 @@
 import Axios, { AxiosResponse, AxiosInstance } from 'axios';
+import { get } from 'lodash';
 import { IApproverInfo, ICanApproveToUser, IRequest } from './approvers.interface';
 import { ApprovalError, NotFoundError, ApplicationError } from '../utils/errors/errors';
-import { config } from '../config';
-import { getToken } from "../spike/spike.service";
+import { config, dests } from '../config';
+import { getToken } from '../spike/spike.service';
 import { TransferRepository } from '../transfer/transfer.repository';
+import { Destination } from '../transfer/transfer.interface';
 
 export class ApprovalService {
+  private instance: AxiosInstance;
 
-    private instance: AxiosInstance;
+  constructor() {
+    this.instance = Axios.create();
+  }
 
-    constructor() {
-        this.instance = Axios.create({ baseURL: config.approval.approvalUrl });
-        this.addAuthIntreceptor();
-    }
-
-    async createRequest(data: IRequest) {
-        try {
-            const res: AxiosResponse = await this.instance.post(`/api/v1/request`, data);
-            return res.data;
-        } catch (err) {
-            await TransferRepository.deleteByID(data.id || "");
-            if (!err.response || !err.response.status) throw new ApplicationError();
+  /**
+   * Create new transfer request in the approval service
+   * @param data - reuqest data
+   * @param destination - approval destination system
+   */
+  async createRequest(data: IRequest, destination: Destination) {
```

Comment:
Add @returns, even if the type is dependant on dropbox. Does it return the new request? 
If there is a type, add it to the function signature.

Reply:
returns nothing